<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="P2ELT22" failures="4" tests="23" name="Test" time="16.839" errors="0" timestamp="2023-10-06T14:23:32 IST">
  <testcase classname="com.restassured.test.UserTests" name="testCreateUser" time="3.859"/>
  <testcase classname="com.restassured.test.UserTests" name="testVerifyUser" time="1.892"/>
  <testcase classname="com.restassured.test.UserTests" name="testLogin" time="1.223"/>
  <testcase classname="com.restassured.test.UserTests" name="testKycUpdate" time="1.736"/>
  <testcase classname="com.restassured.test.UserTests" name="testProfileView" time="0.329"/>
  <testcase classname="com.restassured.test.UserTests" name="testCreateFile" time="1.371"/>
  <testcase classname="com.restassured.test.UserTests" name="testgetFileCertificate" time="0.302"/>
  <testcase classname="com.restassured.test.UserTests" name="testcreateFolder" time="0.298"/>
  <testcase classname="com.restassured.test.UserTests" name="testMoveFile" time="0.295"/>
  <testcase classname="com.restassured.test.UserTests" name="testgetFileVersion" time="0.258"/>
  <testcase classname="com.restassured.test.UserTests" name="createVersion1" time="0.421"/>
  <testcase classname="com.restassured.test.UserTests" name="createVersion1" time="0.793"/>
  <testcase classname="com.restassured.test.UserTests" name="testSendCertificatetoRecycleBin" time="0.291">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.restassured.test.UserTests.testSendCertificatetoRecycleBin(UserTests.java:444)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testSendCertificatetoRecycleBin -->
  <testcase classname="com.restassured.test.UserTests" name="testrenameFolder" time="0.233"/>
  <testcase classname="com.restassured.test.UserTests" name="testdeleteFolder" time="0.247">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.restassured.test.UserTests.testdeleteFolder(UserTests.java:493)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testdeleteFolder -->
  <testcase classname="com.restassured.test.UserTests" name="testRestoreFileFromRecycleBin" time="0.514">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.restassured.test.UserTests.testSendCertificatetoRecycleBin(UserTests.java:444)
at com.restassured.test.UserTests.testRestoreFileFromRecycleBin(UserTests.java:508)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testRestoreFileFromRecycleBin -->
  <testcase classname="com.restassured.test.UserTests" name="testDeleteFilePermanently" time="0.2"/>
  <testcase classname="com.restassured.test.UserTests" name="testgetAllFolders" time="0.215"/>
  <testcase classname="com.restassured.test.UserTests" name="testDeleteFileVersion" time="0.228"/>
  <testcase classname="com.restassured.test.UserTests" name="testRestoreFileVersion" time="0.173"/>
  <testcase classname="com.restassured.test.UserNegativeTests" name="createUser" time="0.214"/>
  <testcase classname="com.restassured.test.UserNegativeTests" name="loginNegative" time="0.68">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="Missing property in path $[&amp;apos;result&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: Missing property in path $['result']
]]>
    </failure>
  </testcase> <!-- loginNegative -->
  <testcase classname="com.restassured.test.UserNegativeTests" name="createfileNegative" time="0.338"/>
</testsuite> <!-- Test -->
